### ISSUE TITLE
Propagate status for Ingress from the child/new ingress created.

### ISSUE THREAD

**Issue by bvennam** [knative-sandbox/async-component#21]

**Comment by beemarie** [comment]
In ingress.go, we manually mark the async ingress as ready. Instead, we should propagate the status from the created (istio) ingress. 

Once the istio ingress is ready, we should mark the async ingress as ready. Since we don't use the async ingress for anything except as a base to the newly created ingress, this status may not even be required.

**Comment by maximilien** [comment]
Duplicate of #32

/close
